manifests/init.pp

```diff
@@ -4,7 +4,7 @@
 #
 # === Parameters:
 #
-# $parent_fqdn::                        FQDN of the parent node.
+# $parent_fqdn::                        FQDN of the parent pulp node.
```

Comment:
We really need a better term than Pulp node given that is/was an actual concept that we are no longer using.

Reply:
Ideas:

 1) Parent pulp and child pulp
 2) Master pulp and pulp
 3) Parent pulp and pulp
 4) Master pulp and replica pulp

Reply:
It's only used in a pulp node situation to point to the qpid hub. Would ```qpid_hub_fqdn``` make sense?

Reply:
+1 but a slight change to `qpid_router_hub_fqdn` or maybe `qpid_router_hub_addr` to match the param? Then again, looking at (https://github.com/Katello/puppet-foreman_proxy_content/blob/c53398bb2c0e697c6921fcc2f78c23b6b99a81be/manifests/dispatch_router.pp#L45) this seems to already exist and could just be used?

Reply:
That's currently used in the pulp master situation to listen on. I'm not sure if we should combine two opposite behaviors into one parameter.

Reply:
Isn't that the point of configuration? The parameter has the same meaning in both situations, what address should the hub be connecting/talking/listening on? In one case, you want to configure it for the server itself, in another you want it to be pointed at another. If we want to two params, I think we gotta make it clear that is the qpid routers 'parent' or 'master' you want to specify.